cli/cli/commands/enclave/inspect/user_services.go

```diff
@@ -49,8 +44,8 @@ func printUserServices(ctx context.Context, _ *kurtosis_context.KurtosisContext,
 			uuidToPrint = uuidStr
 		}
 
-		serviceStatus := userService.GetServiceStatus()
-		serviceStatusStr := service_status_stringifier.ServiceStatusStringifier(serviceStatus)
+		containerStatus := userService.GetContainer().GetStatus()
```

Comment:
nice. If our notion of service status is 1:1 with the underlying container engine - i wonder if we really even need our own `ServiceStatus`.

Reply:
Yeah, I think our own service status can be useful in a management UI where 

Our ServiceStatus - is the intended state
ContainerStatus - is the real state